static/js/common/DropdownMenu.jsx

```diff
@@ -40,7 +45,7 @@ const DropdownMenuItemWithIcon = ({icon, textEn, textHe}) => {
   );
 }
 
-const DropdownMenu = ({children}) => {
+const DropdownMenu = ({children, menuIconComponent}) => {
```

Comment:
What are you passing in here? another react component as a prop? thats generally not considered a good practice...

Reply:
If you need to be able to insert a dynamic component the way to do it is composability, as we have done with the rest of the menu. 
Further break down the top components so you can render the menu icon component inline as children. 
For example 

```
  <DropdownMenu>
     <ProfilePic>
     <DropDownMenuContents>
        <DropDownMenuItem>
        .....
     </DropDownMenuContents>
 </DropdownMenu>
 ````

Reply:
Thanks for pointing this out! I didn't realize, and really appreciate learning about better React practices.

I will push back a drop - in this case, I needed parity with the code that renders the "initials" when a profile image is not present, and needed to structure the code this way iirc to enable that in both PRs. Happy to discuss more, and hear your insights as to if there's a better and more proper React way of doing this. Thanks!